assets/js/modules/thank-with-google/components/setup/SetupPublicationPendingVerification.js

```diff
@@ -39,17 +39,17 @@ export default function SetupPublicationPendingVerification() {
 				'google-site-kit'
 			) }
 		>
-			<Link
+			<Button
 				href="https://publishercenter.google.com/"
-				external={ true }
+				target="_blank"
 				aria-label={ __(
 					'Check your status on Thank with Google Publisher Center',
 					'google-site-kit'
 				) }
-				hideExternalIndicator={ true }
+				inverse={ true }
```

Comment:
We prefer to use the shorthand for boolean `true` props, so let's do the same here (I'm surprised we don't have a lint rule for this 🤔)

```suggestion
				inverse
```

Reply:
Ah, one more preference noted and I'll keep it in mind for future cases. Thank you for going ahead and applying @aaemnnosttv!